book/forms.rst

```diff
@@ -175,7 +175,7 @@ That's it! Just three lines are needed to render the complete form:
     Renders all the fields, which includes the field element itself, a label
     and any validation error messages for the field.
 
-``form_end()``
+``form_end(form)``
```

Comment:
there is no argument in ``form_end()``. You can pass one because Twig ignores extra args, as done by PHP, but your documentation now tells that the argument is needed